src/theme/colors.ts

```diff
@@ -0,0 +1,320 @@
+// Based mostly on https://github.com/Uniswap/interface/blob/main/src/theme/index.tsx
+
+import { opacify } from './colorsUtil'
+
+export interface GlobalPalette {
+  white: string
```

Comment:
can you add a type for Color and then change all the string to Color? similar to the deprecated version